website/eslint.config.js

```diff
@@ -0,0 +1,146 @@
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import react from 'eslint-plugin-react';
+import parser from '@typescript-eslint/parser';
+import eslintPluginAstro from 'eslint-plugin-astro';
+import astroParser from 'astro-eslint-parser';
+import importPlugin from 'eslint-plugin-import';
+import reactHooks from 'eslint-plugin-react-hooks';
+
+const importRules = {
+    'import/no-cycle': 'error',
+    'import/no-deprecated': 'error',
+    'import/no-extraneous-dependencies': 'off',
```

Comment:
I think this one might be useful.
```suggestion
        'import/no-extraneous-dependencies': 'error',
```